### DISCUSSION TITLE
Where to get ada (aka "Buy ada"-button)

### DISCUSSION THREAD

**Discussion by katomm** [cardano-foundation/cardano-org#52] [Ideas]
I would like to take up this idea, posted in a funny way by Rick, and put up for discussion what something like this could **actually look like**.

Tweet was: https://twitter.com/RichardMcCrackn/status/1783827381821866060

![buy ada](https://github.com/cardano-foundation/cardano-org/assets/31965230/47310f5a-d7c0-4206-a181-2b0dec67f8f2)

Currently we link from https://cardano.org/what-is-ada (learn -> what is ada?) to just coin market cap. This seems super outdated. We want people to know about dexes, cexes, banks and other on ramps (earn ada?), staking might fit as well.

![buy ada2](https://github.com/cardano-foundation/cardano-org/assets/31965230/1be5d8b2-1d9c-493d-8e4b-9ea043475e71)

**Comment by RayTracedObserver** [comment]
An educational page dedicated to learning how to purchase ADA and explaining the various options to novice users would indeed be useful.

**Comment by katomm** [comment]
First iteration of this with #83  available now on https://cardano.org/where-to-get-ada
